divvy/cloudfilebrowser.js

```diff
@@ -828,26 +842,68 @@ var cloudFileBrowser = (function() {
             // for each service installed
 
             var tabsHTML = '',
-                containerHTML = '';
-
-            for (var i=0; i<services.length; i++) {
-                tabsHTML += '<li class="' + services[i] + (i == 0 ? ' on' : '' )+ '"><img src="' + servicesImages[i] + '">' + servicesDisplay[i] + '</li>';
-                if (services[i] !== 'onedrivebusiness') {
-                    containerHTML +=    '<div class="' + services[i] + (i == 0 ? ' on' : '' ) + ' drop-zone" aria-element="' + services[i] + '">'+
+                containerHTML = '',
+                sortedServices = services.sort(function(a, b){
+                    return a.order - b.order;
+                });
+            sortedServices.map(function (service, index) {
+                var serviceName = service.service;
+                tabsHTML += '<li class="' + serviceName + (service.order == 0 ? ' on' : '' ) + '"><img src="' + service.image + '">' + service.displayName + '</li>';
+                if (serviceName !== 'onedrivebusiness' && serviceName !== 'sharepoint') {
+                    containerHTML +=    '<div class="' + serviceName + (index == 0 ? ' on' : '' ) + ' drop-zone" aria-element="' + serviceName + '">'+
                     '<h2></h2>' +
```

Comment:
lol wut

Reply:
IDK. This been here forever :P. I don't want to change it.